app/models/call_flow_logic/ews_laos_registration.rb

```diff
@@ -0,0 +1,267 @@
+module CallFlowLogic
+  class EWSLaosRegistration < Base
```

Comment:
It's pretty much similar to Cambodia, but still it's simpler because it has only one language.

I saw the Google sheet, so we used that for our initial communication and mapping audio files.